compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaSymbol.java

```diff
@@ -70,18 +76,18 @@ protected BallerinaSymbol(String name, SymbolKind symbolKind, BSymbol symbol, Co
     public Optional<String> getName() {
         // In the langlib context, reserved keywords can be used as regular identifiers. Therefore, they will not be
         // escaped.
-        if (this.escapedName != null) {
-            return Optional.of(this.escapedName);
-        }
```

Comment:
Removing this means the name will be computed each time `getName()` is called. What's the reasoning behind the change in this section?